app/azure/Dashboard/Dashboard.test.tsx

```diff
@@ -0,0 +1,144 @@
+import { renderWithQueryClient, within } from '@/react-tools/test-utils';
```

Comment:
@itsconquest file should be named `DashboardView/DashboardView.test.tsx`

the template is:
```
ComponentName/index.ts
ComponentName/ComponentName.tsx
ComponentName/ComponentName.test.tsx
ComponentName/ComponentName.stories.tsx (if needed)
ComponentName/ComponentName.module.css (if needed)
```

Reply:
good to know thanks!